modules/sftp-transfer-server/iam.tf

```diff
@@ -0,0 +1,63 @@
+data "aws_iam_policy_document" "transfer_server_assume_role" {
+  statement {
+    effect  = "Allow"
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["transfer.amazonaws.com"]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "transfer_server_assume_policy" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "s3:DeleteObject",
+      "s3:DeleteObjectVersion",
+      "s3:GetBucketLocation",
+      "s3:GetObject",
+      "s3:GetObjectVersion",
+      "s3:ListBucket",
+      "s3:PutObject",
+    ]
+
+    resources = [
+      var.bucket_arn,
+      "${var.bucket_arn}/*"
+    ]
+  }
+}
+
+data "aws_iam_policy_document" "transfer_server_to_cloudwatch_assume_policy" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+    ]
+
+    resources = ["*"]
```

Comment:
This should not be "*" but rather a specific resource or at least a prefix

Reply:
Corrected with a prefix